src/managers/GuildMemberManager.js

```diff
@@ -205,6 +205,24 @@ class GuildMemberManager extends CachedManager {
     return data.reduce((col, member) => col.set(member.user.id, this._add(member, cache)), new Collection());
   }
 
+  /**
+   * Options used for listing guild members.
+   * @typedef {Object} GuildListMembersOptions
+   * @property {Snowflake} [after] Limit fetching members to those with an id greater than the supplied id
+   * @property {number} [limit=1] Maximum number of members to list
+   * @property {boolean} [cache=true] Whether or not to cache the fetched member(s)
+   */
+
+  /**
+   * Lists up to 1000 members of the guild.
+   * @param {GuildListMembersOptions} [options] Options for listing members
+   * @returns {Promise<Collection<Snowflake, GuildMember>>}
+   */
+  async list({ after, limit = 1, cache = true } = {}) {
```

Comment:
Shouldn't the library let Discord decide the default value for `limit`, like it's done for `after`? Just in case, but even if unlikely, Discord changes it.

Reply:
I based the format off of the other methods like [search](https://discord.js.org/#/docs/main/master/typedef/GuildSearchMembersOptions), even though discord does have a limit for it, discord.js previously used the default value

Reply:
Oh right, let's stay consistent then 👌